app/namespace.go

```diff
@@ -1393,15 +1386,10 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 				},
 				&cli.StringFlag{
 					Name:     namespaceRegionFlagName,
-					Usage:    "The region to failover to",
+					Usage:    "The new region to failover. Please use format <cloud provider>-<region name>.",
```

Comment:
```suggestion
					Usage:    "The new region to failover. Please use format <cloud provider>:<region name>.",
```

Reply:
This is the new cloud api format. I cannot change it.